Makefile

```diff
@@ -94,7 +95,7 @@ clean-acceptance:
 build: $(all_modules)
 build-tests: build-test
 build-test: $(addprefix test_,$(go_modules))
-build-all: generate-openapi-generated-clients-and-servers build build-test build-test-app ## Build all modules and tests
+build-all: generate-openapi-generated-clients-and-servers build build-test build-test-app build-gorouterproxy
```

Comment:
How about moving the build-gorouterproxy in the test targets as it just used in int. tets ?

Reply:
good idea, fixed here: 7d48439fa